keras_cv/layers/preprocessing/random_rotation.py

```diff
@@ -138,7 +140,6 @@ def augment_bounding_boxes(self, bounding_boxes, transformation, **kwargs):
         image = None
```

Comment:
can be shortened to:

image = kwargs.get("image", None)

Reply:
Done